src/drivers/mediatek/mt8195/afe-dai.c

```diff
@@ -0,0 +1,123 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
```

Comment:
I think you can make it clear that AFE is an Audio Digital Interfacein the comments or commit message and change the file name and functions prefixes from afe-dai to just `afe`.